### ISSUE TITLE
Support for infinite scrolling

### ISSUE THREAD

**Issue by redsanso** [mattcolman/phaser-list-view#17]
It would be very nice to add a configuration option to let culled items appear to the opposite side of the list (i.e., if I have 5 items, scrolling down after the 5th i would see my list restarting from 1st, 2nd, etc...)

**Comment by mattcolman** [comment]
yeah I see the value for a horizontal scroller, not sure I have time to implement it but happy to review a pull request.
